frontend/src/e2e-playwright/pages/employee/vasu/vasu.ts

```diff
@@ -0,0 +1,65 @@
+// SPDX-FileCopyrightText: 2017-2021 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+import { Page } from 'playwright'
+import { RawElement } from '../../../utils/element'
+
+export default class VasuPage {
+  constructor(private readonly page: Page) {}
+
+  readonly #followupQuestion = this.page.locator(
+    '[data-qa="vasu-followup-question"]'
+  )
+  readonly #finalizeButton = new RawElement(
```

Comment:
Use locators instead of `RawElement` in new code

Reply:
I went ahead and added a deprecated flag on `RawElement` to avoid this in the future.